libs/wingsdk/src/std/array.ts

```diff
@@ -311,4 +322,15 @@ export class MutArray {
     value;
     throw new Error("Macro");
   }
+
+  /**
+   * Returns the array, sorted and also mutated
+   *
+   * @macro $self$.sort()
```

Comment:
@Chriscbr what do you think about passing a sorting function?

Reply:
I thought about a comparator, although JSII does not support generics, so we will need to rely on `any` for the comparator types